### PR TITLE
🔼 Upgrade Ember v3.21.2...v3.26.1

### PR DESCRIPTION
**Pull request by achambers**
This PR does a standard Ember update, using `ember-cli-update --to 3.26`

[CHANGELOG](https://upgrade.emberjs.com/changes?fromVersion=3.21&toVersion=3.26)

### CODE REVIEW

File: addon/components/menu/button.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <button
   {{on "click" @onClick}}
-  {{on "keydown" this.onKeydown}}
+  {{on "keyup" this.onKeyup}}
```
[the remark this answers is not in the document]
Comment (GavinJoyce) [reply addon/components/menu/button.hbs:3]: Oh, I see - to address prettier warnings.

It seems that headlessui uses both, mostly `keydown` though. Perhaps we should disable the prettier rule for now and I'll revisit this as part of my efforts to complete the `<Menu>` component?

[the remark this answers is not in the document]
Comment (achambers) [reply addon/components/menu/button.hbs:3]: Yeh, that was the reason. I’d generally err on the side of not disabling prettier warnings. But happy to for this PR if you want to swing back there at a later date.

[the remark this answers is not in the document]
Comment (GavinJoyce) [reply addon/components/menu/button.hbs:3]: Sounds good, thanks

File: .template-lintrc.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  extends: 'octane'
+  extends: 'octane',
+
+  rules: {
+    'no-down-event-binding': 'warn',
```
Comment (achambers) [inline .template-lintrc.js:7]: @GavinJoyce Turns out it was a template lint error and not prettier. I decided to put this in as a `warn` so it stays on our radar and we can decide what to do at a later date. The error is to help with disability so it feels like maybe it's something we want to consider. But anyway, linting doesn't fail now and your code is back to how it was originally.

**Review by GavinJoyce — APPROVED** [review]
This is great, nice one

**Comment by GavinJoyce** [comment]
It's weird that the PR didn't show a red ❌, the tests seem to be timing out:

Screenshot 2021-05-21 at 08 06 56

**Comment by GavinJoyce** [comment]
Seems to be related to an embroider scenario:

Screenshot 2021-05-21 at 08 10 46

I'm might disable embroider for now, we can add it back when we address the issue

**Comment by GavinJoyce** [comment]
https://github.com/GavinJoyce/ember-headlessui/pull/33

**Pull request merged**